student_s.rb

```diff
@@ -6,6 +6,11 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
## OPTIONAL 
* I strongly suggest that the instance variable `@classroom` has setters as you did for your `classroom` class remember `a student belongs to a classroom` relationship